template-parts/post/author-bio.php

```diff
@@ -8,7 +8,7 @@
  */
 
 ?>
-<div class="author-bio <?php echo get_option( 'show_avatars' ) ? esc_attr( 'show-avatars' ) : ''; ?>">
+<div class="author-bio <?php echo get_option( 'show_avatars' ) ? 'show-avatars' : ''; ?>">
```

Comment:
Good catch, mind fixing https://github.com/WordPress/twentytwentyone/blob/trunk/comments.php#L25 too?

Reply:
Sure, I'll do it first thing tomorrow :) mobile right now

Reply:
Done